### DISCUSSION TITLE
Adding two spaces after punctuation?

### DISCUSSION THREAD

**Discussion by ghost** [SYSTRAN/faster-whisper#160] [Q&A]
The documents I'm trying to produce look better with a double space after a period (old typewriter style).  Just like I'm doing in this text.  I've tried to peer through the transcribe module and the tokenizer, but I can't seem to find where this behavior is actually controlled.  Is this part of the model used, faster-whisper, or CTranslate2?

**Comment by guillaumekln** [comment]
The end of a sentence is not clearly defined anywhere. You would need to postprocess the text transcription with your own sentence segmentation logic.
